LinqOverWCF/Windows Host/Client/Client.csproj

```diff
@@ -1,5 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
yuck, legacy project format. Would be nice to migrate it to CPS format, shouldn't be a lot of work